pkg/cli/kubectlwrappers/wrappers.go

```diff
@@ -226,16 +225,7 @@ func NewCmdEdit(fullName string, f kcmdutil.Factory, streams genericclioptions.I
 
 // NewCmdConfig is a wrapper for the Kubernetes cli config command
 func NewCmdConfig(fullName, name string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
-	pathOptions := &kclientcmd.PathOptions{
-		GlobalFile:       kclientcmd.RecommendedHomeFile,
-		EnvVar:           kclientcmd.RecommendedConfigPathEnvVar,
-		ExplicitFileFlag: genericclioptions.OpenShiftKubeConfigFlagName,
```

Comment:
@sallyom this was the place that was panicing for you, b/c it tried to have `--config` registered twice.

Reply:
oh, I see what I was doing wrong now.  I was duplicating the `if !openshiftpatch.IsOC` in this file, so not fixing the panic.